UI/api-interface.cpp

```diff
@@ -67,6 +67,11 @@ struct OBSStudioAPI : obs_frontend_callbacks {
 		return (void*)main->winId();
 	}
 
+	void *obs_frontend_get_system_tray(void) override
+	{
+	    return (void*)main->trayIcon.data();
```

Comment:
You have a tab for an indent, and then 4 spaces for an indent here.  Should be two tabs.

Reply:
Whoops!